src/main/java/com/gooddata/md/MetadataService.java

```diff
@@ -97,6 +101,31 @@ public MetadataService(RestTemplate restTemplate) {
     }
 
     /**
+     * Retrieves a collection of objects corresponding to the supplied collection of URIs.
+     *
+     * @param project project that contains the objects to be retrieved
+     * @param uris collection of URIs
+     * @return collection of metadata objects corresponding to the supplied URIs
+     */
+    public Collection<Obj> getObjsByUris(Project project, Collection<String> uris) {
```

Comment:
Please add `final` to the arguments

Reply:
only one method in that class (out of approx. 20) uses "final" with arguments. should I add it to all?

Reply:
@ostumpf as far as i know we have no consensus about `final` arguments, I personally prefer *without* `final` ;-)

Reply:
Basically I see it that most of "new" methods and services contain final method arguments. There is no consensus about it among Java developers, I personally prefer the variant with final which prevents you from accidentely overwriting the variable. I suggest to add final, but let's not spend more time bickering :)